### ISSUE TITLE
KOALA-specific tutorials + help cubing please, please, please

### ISSUE THREAD

**Issue by angelrls** [pykoala/pykoala#430]
Hi everyone,

I've uploaded 3 new tutorials that consider the KOALA-specific tools, these are in my branch:

1. Intro and generic tasks
2. Obtaining the night calibration
3. RSS processing a science observation

These are the tutorials I'll follow for the training session in 2 weeks (remember that is for KOALA, the users are requesting KOALA-specific tools!). Still, some of these corrections could still be applied to any RSS (just if .koala is replace by .instrument or similar).

1 and 3 are completed, but 2 needs the flux calibration (for which I need cubes of calibration stars)

Tutorial "4. Cube processing and mosaicking" is still to be done...

But again I need help in many aspects:

- I need to download the latest version of the code and test that my tools still work.  I am really afraid of doing this alone/without help as I don't want to delete the version I have now.

- My main problem is that I still can't get the cubing to work. In a moment of despair a month ago, I implemented a rough version of cube objects following the same structure that in main code for testing my tools using my old code, it seemed to work and my tools for plotting and checking ADR and create the flux calibration work there. However, during the weekend trying to make a mosaic, I realised that my cubes were or inverted vertically or rotated 180º... I need to get the cubing working, ideally using the latest version of the code, please, please, please, some help?

- Yes, many things in my tools NEED TO BE CHECKED AND UPDATED PROPERLY WITH NICE PROGRAMMING, and even some things replaced for existing python or astropy tools I don’t know,  but they work and provide good results so far (as they worked in my old version of the code). Particularly, I need to check the tools in onedspec.py, but again that is something we can do AFTER we get the rest working.

- All the updates should go to the main branch on time for the tutorial session in 2 weeks. Again, I'm NOT changing anything in the main code, I'm just adding extra functions (mainly in koala_ifu.py, but they are calling 1D functions in onedspec.py and my main plotting function plot_plot).

Thanks.

**Comment by miguelglezb** [comment]
May I suggest not freezing main completely? I'm doing modifications on the clarity in the documentation based on @angelrls suggestions, as well as for separating the contents for regular users and developers in them.

**Comment by paranoya** [comment]
I don't think we can freeze main yet. Given the timescale, we should have a (potentially long) conversation about the tutorials that will be shown in the training week, and how they integrate within the project.

**Comment by miguelglezb** [comment]
I agree with the (potentially long) conversation!

**Comment by angelrls** [comment]
Thanks. See update in the other issue I created. What I uploaded in this pushed can be ignored, as I have spent all my weekend updating them to the latest version of the code and completing documentation. I only need help uploading to GitHub all those changes for you to see (after all the problems I have had, I don't want to mess up with the branches as I'm unfamiliar to those). @miguelglezb I'll need your help here.

Thanks.
